executor/opensubmit/executor/__init__.py

```diff
@@ -10,9 +10,9 @@
 import time
 from threading import Timer
 from datetime import datetime, timedelta
-from lockfile import FileLock
```

Comment:
What is the problem with the lockfile package? It seems to be the right choice for having cross-platform locking. Some explanation for the "manual" solution would be good.